src/routes/safe/components/AddressBook/CreateEditEntryModal/index.tsx

```diff
@@ -94,7 +94,7 @@ export const CreateEditEntryModal = ({
                         testId={CREATE_ENTRY_INPUT_NAME_ID}
                         text="Name"
```

Comment:
I find some places where an input is mandatory but we are not setting the *, while this character is tipically used for indicating that. I would like to raise this here because it's missing so either we should add it also here as it's a mandatory field or remove it from every place were we have it, because I don't even remember an input that could accept empty values  in the whole app

Reply:
@katspaugh would also like to read your input here

Reply:
Personally not a fan of literally typing `*` in the label.
Ideally the component should indicate somehow that it's required if the prop `required` is true.
But yeah, for consistency with the other inputs it might make sense to add it here.

Reply:
done